test/secmemtest.c

```diff
@@ -7,13 +7,15 @@
  * https://www.openssl.org/source/license.html
  */
 
+#include <../e_os.h>  /* pull in unistd.h for _POSIX_VERSION */
```

Comment:
Please use quotes, as that's a local header only (i.e. will never appear as a system header)

Reply:
yeah, good catch!